ovos_phal_connectivity_events/__init__.py

```diff
@@ -81,6 +113,5 @@ def handle_check(self, message):
         if state != self.state:
             self.update_state(state, message)
 
-        # check again in self.sleep_time
-        time.sleep(self.sleep_time)
-        self.bus.emit(message)
+        if message:
+            self.bus.emit(message)
```

Comment:
this should be removed, it was just to fake the thread behavior after the sleep, we dont want to emit it again. that was just using the events mechanism instead of proper thread

Reply:
I misread this as a reply, I'll remove it